pkg/operation/botanist/waiter.go

```diff
@@ -83,8 +85,13 @@ func (b *Botanist) WaitUntilNoPodRunning(ctx context.Context) error {
 		for _, pod := range podList.Items {
 			if pod.Status.Phase == corev1.PodRunning {
 				b.Logger.Info("Waiting until there are no running pods in the shoot cluster (at least one pod still exists)", "pod", client.ObjectKeyFromObject(&pod))
-				return retry.MinorError(fmt.Errorf("waiting until there are no running Pods in the shoot cluster... "+
-					"there is still at least one running Pod in the shoot cluster: %q", client.ObjectKeyFromObject(&pod).String()))
+
+				message := "waiting until there are no running Pods in the shoot cluster ... there is still at least one running Pod in the shoot cluster: %q"
```

Comment:
Nit: Not sure why there is no an additional space (` `) in the message. I would prefer to remove it again.

That being said, I prefer the log message without the `...`.

Reply:
`...` is addresses, what you you mean with the additional space? I can't seem to find any double whitespaces.

Reply:
There is a space character before `...`. This space was not there before your change.
It is just unusual to me to have a space before a dot.

Reply:
I understand, the log message was kept form the previous implementation. But it should not be an issue now as the `...` is gone.